src/prob/mld.jl

```diff
@@ -4,6 +4,12 @@ function run_mc_mld(data::Union{Dict{String,<:Any},String}, model_type::Type, so
 end
 
 
+"Run multinetwork load shedding problem with storage"
+function run_mn_mc_mld(data::Union{Dict{String,<:Any},String}, model_type::Type, solver; kwargs...)
+    return run_mc_model(data, model_type, solver, build_mn_mc_mld; kwargs...)
```

Comment:
To follow the PowerModels conventions, this call requires `run_model(...; multinetwork=true, kwargs...)`.  The user is not expected to set this one level up.